### ISSUE TITLE
{FEATURE REQUEST} Consecutive Buy/Sell selections to Catch uptrends and Sell at end of Trend

### ISSUE THREAD

**Issue by sanmaccrypto** [taniman/profit-trailer-enhancements#454]
In an effort to trade the bot the way I manually trade, it would be great to be able to select "and", "or" or "then" when selecting buy and sell configuration settings.  If the same could be applied to multiple indicator periods per setting (eg. oversold on both the hourly and 15 min candles) then that would allow us to better trade with the trends.  Currently we have an "and" selection eg. EMA Gain 0.1 and Low BB 20, etc.
If the + or - next to each selection (which currently selects only on or off) for the buy/sell setting was actually an and/or/then selection we could:
- "and" - turns that buy/sell selection on in series with the other selections
- "or" - turns that buy/sell selection on in parallel with the other selections
- "then" turns on each buy/sell selection in consecutive order only after the previous selection triggers.

For example trading on 15 minute candles; to try to catch an oversold coin which has bottomed out on the RSI and stochastic then started to trend back upwards, we could do something like this:
1 hourly candles:  STOCHRSI 20 limit 80 *and* STOCHRSICROSS 0.05
*and*
15 min candles:  RSI below 30, *and* STOCHRSI below 20, *then* STOCHRSICROSS 0.05, *and* STOCHRSI above 15

This should avoid buys when we get a failed oversold STOCHRSICROSS indication as PT will wait for STOCHRSI to increase above 15 before it buys but only after it has crossed in a positive manner and only in a longer term uptrend.   As it has bought on the upswing on both 15 min and 1 hour candles we stand a better chance of trading with the trend.  I may not have explained that the best way but that's the rough idea.

**Comment by Masterofcoinak** [comment]
Possible already https://wiki.profittrailer.com/doku.php?id=pairs.properties#default_buy_strategy_formula
Should be closed

**Comment by sanmaccrypto** [comment]
This is not correct.  Do not close thanks

**Comment by sanmaccrypto** [comment]
do not close.  Feature does not exist.

**Comment by LiamAird** [comment]
Please do explain the further need for this feature request? 
You can use the buy formula and labels to specify different different strategies and different time intervals.

**Comment by sanmaccrypto** [comment]
Hello,

The bot does not currently allow what I suggested.  It is not about setting different buy strategies and labels or time intervals.  It is not about using the same indicator multiple times.  It is not about buy-on-first-signal.

The bot currently only allows buying above or below a set point.  It does not allow buying using consecutive points ie. after trigger point 1 is reached, then look for trigger point 2 (and 3, 4, etc) on the same indicator.  I provided an example in the guthub about this.

Basically my proposal may allow a better way to pick bottoms/tops of trend as opposed to trailing the price.  My proposal’s not exactly trailing the RSI, for example, but performs a similar function.

I have had this discussion a few times now on the support discord and each time I am told that PT does not have this function.

If you are unsure what I mean, please get back to me.  Thanks.

Regards,

Stuart.
